packages/jest-preset-mc-app/has-jsx-runtime.js

```diff
@@ -0,0 +1,14 @@
+function hasJsxRuntime() {
```

Comment:
Just liked to have this separately. We have it once in the for webpack and once for babel as CRA.